applications/nrf_desktop/doc/board.rst

```diff
@@ -0,0 +1,40 @@
+.. _board:
+
+Board module
+############
+
+The ``board`` module is used to ensure the proper state of GPIO pins that are connected to electrical components and not configured by other modules in the application.
```

Comment:
`to ensure the proper state of GPIO pins that are not configured by other modules in the application`

Reply:
I just wanted to mention that pins that are not connected to anything may be left unset. Board module only has to configure pins that are in use (e.g. to reduce power consumption). Still, if you want I may remove this information